console.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python3
+"""Contains a command line interpreter class"""
+import cmd
+import re
+import shlex
+import sys
+from typing import List
+from models import storage
+from models.base_model import BaseModel
+from models.user import User
+from models.state import State
+from models.city import City
+from models.place import Place
+from models.amenity import Amenity
+from models.review import Review
+
+
+class LodgifyCommand(cmd.Cmd):
+    """Class that initializes a cmd interpreter"""
+    prompt = "(lodgify) "
+    classes = [
+        "BaseModel",
+        "User",
+        "State",
+        "City",
+        "Amenity",
+        "Place",
+        "Review"
+        ]
+    objects = storage.all()
+
+
+    def do_quit(self, line: str) -> bool:
+        """command to exit the cmd prompt"""
+        return True
+
+    def default(self, line: str) -> None:
+        return super().default(line)
```

Comment:
Please remove the return statement to prevent code redundancy cause it's specified the code returns None